### ISSUE TITLE
Running tests in parallel slower compared to running individual tests.

### ISSUE THREAD

**Issue by JiaZhenLin** [xunit/xunit#2920]
For ex,I have 2000 tests,And my computer has 8 core,and run 8 tests in parallel.then each test maybe take 2s,if i run individual,it  maybe take 1s for each test.If me sometings woring?How can I speed up the tests  in parallel?

**Comment by bradwilson** [comment]
It would be helpful if you could provide some examples of the kinds of test that are slow, maybe even a repro project.

**Comment by TylerBrinkley** [comment]
It might be related but we just upgraded to xunit 2.8.0 from 2.7.0 and our the test duration nearly doubled and we use a lot of parallelization.

**Comment by bradwilson** [comment]
@TylerBrinkley Have you tried reverting back to the pre-2.8 parallelization algorithm?

The release notes mentioned the new algorithm:

![image](https://github.com/xunit/xunit/assets/16944/9f8a6439-e644-485b-ab21-a92d43866d61)

For more information, see the docs page: https://xunit.net/docs/running-tests-in-parallel#algorithms

**Comment by TylerBrinkley** [comment]
Thanks @bradwilson. I changed the `parallelAlgorithm` from the new default of `conservative` to `aggressive` and the tests are running as quick as they were before. Thanks!

**Comment by bradwilson** [comment]
@JiaZhenLin Does my answer to Tyler help you resolve your issue?

**Comment by bradwilson** [comment]
Hello,

As part of the effort to clean up older issues, this is being closed due to age. If this is still applicable with v3, please open a new issue (with an attached v3 repro) so it can be re-evaluated and categorized.

Thanks!
